pandas/core/frame.py

```diff
@@ -907,19 +910,17 @@ def itertuples(self, index=True, name="Pandas"):
 
         Examples
         --------
-
         >>> df = pd.DataFrame({'col1': [1, 2], 'col2': [0.1, 0.2]},
-                              index=['a', 'b'])
+        ...                   index=['a', 'b'])
```

Comment:
In general I think it's better to have data that somehow looks real. I think it helps understand better, as you don't need to check in the DataFrame constructor that the row a, in the col2, has a 0.1. It's self-contained that a cat has 4 legs (see this example: https://github.com/pandas-dev/pandas/blob/master/pandas/core/frame.py#L7580).

Also, if you can also one examples with `index=False`, and one with `name='Animal` (or whatever concept you think it's better), that would also be valuable.

Reply:
Awesome. I really like the animal ideas, it is super intuitive! 

I made the changes as such and will push it up now. 

```
        Examples
        --------
        >>> df = pd.DataFrame({'num_legs': [4,2], 'num_wings': [0,2]},
        ...                   index=['dog','hawk'])
        >>> df
              num_legs  num_wings
        dog          4          0
        hawk         2          2
        >>> for row in df.itertuples():
        ...     print(row)
        ...
        Pandas(Index='dog', num_legs=4, num_wings=0)
        Pandas(Index='hawk', num_legs=2, num_wings=2)

        By setting the `index` parameter to False we can remove the index
        as the first element of the tuple:

        >>> for row in df.itertuples(index=False):
        ...     print(row)
        ...
        Pandas(num_legs=4, num_wings=0)
        Pandas(num_legs=2, num_wings=2)

        With the `name` parameter set we set a custom name for the yielded
        namedtuples:

        >>> for row in df.itertuples(name='Animal'):
        ...     print(row)
        ...
        Animal(Index='dog', num_legs=4, num_wings=0)
        Animal(Index='hawk', num_legs=2, num_wings=2)
```